src/HTSPDemuxer.cpp

```diff
@@ -247,6 +247,26 @@ PVR_ERROR CHTSPDemuxer::CurrentSignal ( PVR_SIGNAL_STATUS &sig )
   return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR CHTSPDemuxer::CurrentDescrambleInfo ( PVR_DESCRAMBLE_INFO &info )
+{
+  CLockObject lock(m_mutex);
+
+  info = {0};
```

Comment:
Ah, and main reason is, if a member gets added to a struct by an api change, this prevents that the initialization of the new member must be explicitly added to every addon. As long as the new member's default value is "0" (in memory represantation), this is very handy.